cases/detector/api_lifecycle_policy_v1beta1.go

```diff
@@ -0,0 +1,68 @@
+package detector
+
+import (
+	"strconv"
+
+	"github.com/kakao/detek/cases/collector"
+	"github.com/kakao/detek/pkg/detek"
+	"k8s.io/api/policy/v1beta1"
+)
+
+var _ detek.Detector = &ApiLifecyclePolicyV1Beta1{}
+
+type ApiLifecyclePolicyV1Beta1 struct{}
+
+func (*ApiLifecyclePolicyV1Beta1) GetMeta() detek.DetectorInfo {
+	psp := &v1beta1.PodSecurityPolicy{}
+	deprecatedMajor, deprecatedMinor := psp.APILifecycleDeprecated()
+	removedMajor, removedMinor := psp.APILifecycleRemoved()
+
+	return detek.DetectorInfo{
+		MetaInfo: detek.MetaInfo{
+			ID:          "api_lifecycle - policy/v1beta1",
+			Description: "Finding api that is unavailable in the future Kubernetes releases.",
+			Labels:      []string{"kubernetes", "api"},
+		},
+		Required: detek.DependencyMeta{
+			collector.KeyK8sPolicyV1Beta1PodSecurityPolicyList: {Type: detek.TypeOf(v1beta1.PodSecurityPolicyList{})},
+		},
+		Level: detek.Warn,
+		IfHappened: detek.Description{
+			Explanation: "The policy/v1beta1 PodSecurityPolicy is deprecated in v" + strconv.Itoa(deprecatedMajor) + "." + strconv.Itoa(deprecatedMinor) + "+, " +
+				"unavailable in v" + strconv.Itoa(removedMajor) + "." + strconv.Itoa(removedMinor) + "+",
```

Comment:
[Optional]
Although they have the same functionality, I believe the following code is more readable:
```go
fmt.Spritf("The PodSecurityPolicy in policy/v1beta1 is deprecated in %s, and unavailable in %s",
  fmt.Spritf("v%d.%d", deprecatedMajor, deprecatedMinor),
  fmt.Spritf("v%d.%d", removedMajor, removedMinor),
)
```